### PR TITLE
Added Game Controller icon & Horse Gallop functionality.

### PR DESCRIPTION
**Pull request by adsingh14**
Press `Shift` then `Arrow Keys` to make horse gallop.

Todo
- [ ] Horse not galloping after releasing 'Shift' key but walks.

### CODE REVIEW

File: public/scripts/main.js [hunk 1]
```diff
@@ -62,38 +59,59 @@ function changeCursor(elem,type) {
...
+*   Author - @adsingh14
+*/
+
+let clap = 1;
+const clapBtn = document.getElementById("clapBtn"),
+clapCounter = document.querySelector('.clapCounter');
+
+clapBtn.addEventListener('click', function(){
+    clapCounter.textContent = clap ++;
+    setTimeout(function () {
```
Comment (adsingh14) [inline public/scripts/main.js]: It'd be great if you remove whitespace between function & it's parenthesis. 
❌: `function ()`
✔️ : `function()`

**Comment by caseydierking** [comment]
@adsingh14 , can you branch off of current master and add in the horse galloping functionality?

It seems like somewhere your icons may have gotten a bit messed up. They are all fuzzy. Also, I'd say go ahead and remove the controller icon. We have some other thoughts on what we want to do with that button and will be updating the readme shortly to clarify that.

I do think the galloping feature is great though. So perhaps just pulling that in in a separate pull request and we can close this one. Thoughts?

**Comment by adsingh14** [comment]
>can you branch off of current master and add in the horse galloping functionality?
- Okay, I will !

>It seems like somewhere your icons may have gotten a bit messed up. 
- Seems fine to me. That's why I added **.svg** . Yes, you can share your thoughts on that.

**Comment by adsingh14** [comment]
@caseydierking Created new branch for **Horse Gallop** - https://github.com/caseydierking/wcss/pull/92

**Pull request closed without merging**